state-chain/runtime/src/chainflip/solana_elections.rs

```diff
@@ -227,11 +234,22 @@ impl
 		alt_identifier: SolanaAltWitnessingIdentifier,
 		alts: AltConsensusResult<Vec<SolAddressLookupTableAccount>>,
 	) {
-		Environment::add_sol_ccm_swap_alts(alt_identifier.swap_request_id, alts);
-	}
-
-	fn should_expire_election(alt_identifier: SolanaAltWitnessingIdentifier) -> bool {
-		crate::System::block_number() >= alt_identifier.election_expiry_block_number
+		// use set_unsynchronised_state_map to write result onto the state map
+		match alts {
+			AltConsensusResult::ValidConsensusAlts(res) => res.into_iter().for_each(|r| {
```

Comment:
res implies result, but it's actually just lookup tables here
```suggestion
			AltConsensusResult::ValidConsensusAlts(lookup_tables) => lookup_tables.into_iter().for_each(|lookup_table| {
```